### PR TITLE
Use latest FluentUI Blazor packages

### PR DESCRIPTION
**Pull request by vnbaaij**
Update `Directory.Build.props` to use latest 4.7.0 Fluent UI Blazor packages
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3884)

### CODE REVIEW

**Review by danmoseley — APPROVED** [review]

File: Directory.Packages.props [hunk 1]
```diff
@@ -98,8 +98,8 @@
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="KubernetesClient" Version="13.0.26" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.6.1" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.6.1" />
+    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.7.0" />
```
Comment (danmoseley) [inline Directory.Packages.props:101]: Looks like one is already 4.7.1?

**Review by danmoseley — COMMENTED** [review]
Dependabot I guess didn't run yet

**Comment by danmoseley** [comment]
Btw just curious, apparently you're mirroring these, or they wouldn't restore? (Which is good you can do it yourself)

File: Directory.Packages.props (hunk 1 above)
Comment (vnbaaij) [reply Directory.Packages.props]: Yes, found an issue in 4.7.0 and published new NuGet couple of hours later. PR now updated for this

**Comment by danmoseley** [comment]
Looks like this update already happened 🙂

**Pull request closed without merging**